### PR TITLE
Add Intel UP2 support with CentOS 7

### PR DESCRIPTION
**Pull request by ctron**
Add support for Intel UP² based on CentOS 7

**Note:** This PR also requires #2091 to be fixed. Possibly by merging #2092

### CODE REVIEW

File: kura/distrib/src/main/resources/intel-up2-centos-7/kura.properties [hunk 1]
```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (c) 2011, 2017 Eurotech and others
```
Comment (MMaiero) [inline kura/distrib/src/main/resources/intel-up2-centos-7/kura.properties]: 2018?

File: kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh [hunk 2]
```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+#
+# Copyright (c) 2016 Red Hat Inc and others
```
Comment (MMaiero) [inline kura/distrib/src/main/resources/intel-up2-centos-7/kura_install.sh]: 2018?

Same for other files contributed here.

**Comment by MMaiero** [comment]
@dwoodard1 I don't have a board to do some testing. Could you do a check with yours?

**Comment by ctron** [comment]
@MMaiero I added 2018 to the copyright headers. But only when they existed and I actually modified the file.

**Comment by MMaiero** [comment]
@ctron Do you have any hurry on this? I would like to keep on hold until we get feedback from Dave. Just to verify that everything works as expected with a proper QA validation.

**Comment by ctron** [comment]
I don't see anything which speaks against merging that now. It doesn't change Kura itself, just adds a new distribution profile. And letting the PR wait and longer, only causes more trouble later on when it has to be re-based and possible conflicts being resolved.

And I also don't really understand what you mean by proper QA validation at this point. That is normally done right before the release of Kura. And when Kura enters that phase, then changes like that won't get merged as they are deemed too risky to be merged. At least that was the argument in the past.

So unless anything speaks against merging this, then I would say, merge it now.

**Comment by dwoodard1** [comment]
Sorry, just now getting around to this. Question, why are we targeting CentOS? If I'm not wrong, the UP2 comes with Ubuntu (at least mine did). Isn't installing CentOS before installing Kura an unnecessary step?

**Comment by ctron** [comment]
Well you may be targeting Ubuntu as well. But my focus is on targeting CentOS. And I think in the spirit of being open source, there is nothing wrong with that approach.

**Comment by dwoodard1** [comment]
Of course I have no issue supporting CentOS. I think it is a nice addition. I just didn't understand starting here, since unlike the Raspberry Pi, the UP2 ships with an installed OS. I will create a separate PR for Ubuntu support, since I assume that will be the simplest path for new Kura/UP2 users.

Since this doesn't impact the framework directly, only adds a new distribution, I think we can merge. If we find issues we can create separate issues. I can test this after I have tested the default Ubuntu installation. @ctron did you want to make any changes before I merge?

**Comment by ctron** [comment]
Agreed. No please go ahead and merge. As you said, if we find issues we can always improve.

**Pull request merged**